src/main/java/fr/uge/chargepointconfiguration/chargepoint/ocpp/ocpp2/data/SetVariableDataType.java

```diff
@@ -0,0 +1,13 @@
+package fr.uge.chargepointconfiguration.chargepoint.ocpp.ocpp2.data;
+
+/**
+ * Defines the type used in the ChangeConfigurationSRequest.
+ *
+ * @param attributeValue The given value to the variable.
+ * @param component The variable's component.
+ * @param variableType The component.
+ */
+public record SetVariableDataType(String attributeValue,
```

Comment:
is it like a (type : key -> value) triplet for the configuration of the fields ?

Reply:
According to the documentation, yes !
AttributeValue is the value
component is the key
variableType is the type.

This is more clear on this class, like you've said it is a triplet so the javadoc should tell you that component is the key

Reply:
Deleted